include/camp/resource.hpp

```diff
@@ -112,11 +136,13 @@ namespace resources
       };
 
       template <typename T>
-      class ContextModel : public ContextInterface
+      class ContextModel final : public ContextInterface
       {
       public:
         ContextModel(T const &modelVal) : m_modelVal(modelVal) {}
         Platform get_platform() const override { return m_modelVal.get_platform(); }
+        bool operator==(Resource const& r) override { return m_modelVal == r.get<T>(); }
```

Comment:
Adding a blank line to separate related methods from others helps readability. For example, `==` and `!=` are related, `allocate`, `calloc`, and `deallocate` are related, etc.